src/constants/apiPaths.ts

```diff
@@ -1,5 +1,6 @@
 const API_PATHS = {
-  product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+  product:
+    'https://hzpkuynm2m.execute-api.eu-west-1.amazonaws.com/dev/products',
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://i2ahpk73lj.execute-api.eu-west-1.amazonaws.com/dev',
   bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
```

Comment:
Probably you should also update "bff" path, because "Manage products" page doesn't show products:

![image](https://user-images.githubusercontent.com/22929109/181208799-c5ffa049-7160-41b3-a26f-f6d4af21b7c8.png)